CMakeLists.txt

```diff
@@ -93,11 +88,19 @@ else()
 endif()
 
 add_library(onnx_proto ${PROTO_SRCS} ${PROTO_HDRS})
-add_dependencies(onnx_proto protobuf::libprotobuf)
-target_compile_options(onnx_proto PRIVATE /WX- ${DLLEXPORT_OPTION})
+target_include_directories(onnx_proto PUBLIC "${PROTOBUF_INCLUDE_DIRS}")
+target_link_libraries(onnx_proto ${PROTOBUF_LIBRARIES})
```

Comment:
Should be `target_link_libraries(onnx_proto PUBLIC ${PROTOBUF_LIBRARIES})`

Reply:
thanks. fixed. Just curious, what's the default value?